src/L2GovernorMetadata.sol

```diff
@@ -17,6 +18,26 @@ abstract contract L2GovernorMetadata {
   /// @notice The id of the proposal mapped to the proposal metadata.
   mapping(uint256 proposalId => Proposal) _proposals;
 
+  /// @notice The assumed block time of the base network
+  uint256 private L1_BLOCK_TIME = 12;
+  /// @notice The assumed block time of the aux network
```

Comment:
```suggestion
  /// @notice The assumed block time of the target network
```

Reply:
Done in 165e503186e3748ec195858fcb3972d1021c0610